fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -1729,7 +1751,9 @@ struct MetaclusterManagementWorkload : TestWorkload {
 				dataClusterChecks.push_back(checkDataCluster(self, clusterName, dataClusterData));
 			}
 			auto capacityNumbers = metacluster::util::metaclusterCapacity(dataClusters);
-			ASSERT(capacityNumbers.first.numTenantGroups == self->totalTenantGroupCapacity);
+			// The number of tenant groups on each cluster can be greater than the capacity limit due to the
+			// `ignoreCapacityLimit' configuration option.
+			ASSERT(capacityNumbers.first.numTenantGroups >= self->totalTenantGroupCapacity);
```

Comment:
The way the test previously accounted for this was to let the tenant group capacity be the large of a cluster's capacity and the number of assigned tenant groups. This is because it was previously possible to end up with too many tenant groups in the case that you changed the capacity to a value lower than what is already there.

Reply:
Done